PR2457-Observability Rel 2.1 Hilla (#2498)
vaadin/docs: articles/lit/acceleration-kits/observability-kit/configuration.adoc

The service attributes are as follows:

> - `service.name` is an attribute that's used to distinguish a service by name. The default value is `vaadin`. It's the only attribute with a default value and the only one required.

Reviewer: Considering that the attribute has a default value, perhaps we can drop the required bit, like this:
```suggestion
- `service.name` is an attribute that's used to distinguish a service by name. It's the only attribute with a default value, which is `vaadin`.
```
